zebra-state/src/on_disk.rs

```diff
@@ -38,33 +38,23 @@ impl SledState {
         let hash: BlockHeaderHash = block.as_ref().into();
         let height = block.coinbase_height().unwrap();
 
-        let by_height = self.storage.open_tree(b"by_height")?;
+        let height_map = self.storage.open_tree(b"height_map")?;
         let by_hash = self.storage.open_tree(b"by_hash")?;
 
         let mut bytes = Vec::new();
         block.zcash_serialize(&mut bytes)?;
 
         // TODO(jlusby): make this transactional
```

Comment:
I don't think we actually want a transaction here, because we can add blocks with a hash, without adding them to the main chain heights.